components/style/properties/properties.mako.rs

```diff
@@ -2115,6 +2115,14 @@ pub fn apply_declarations<'a, F, I>(device: &Device,
     }
 
     {
+        use computed_values::display::T as display;
+        if context.layout_parent_style.writing_mode != style.writing_mode &&
```

Comment:
Let's add here:

 * A quote and link to the spec.
 * A link to Boris' www-style mail.
 * (optionally) A link to the issue we're fixing to see the discussion.